agc_environment.yml

```
@@ -0,0 +1,11 @@
+name: agc
+channels:
+  - bioconda
+  - conda-forge
+dependencies:
+  - git
+  - make
+  - go
+  - go-cgo
+  - nodejs
+  - python >=3.9,<3.10
```

AGC require the python3.9 (https://github.com/SequenceBio/amazon-genomics-cli/blob/main/packages/wes_adapter/install.sh), so I request you to strict the python version. 
Build also require **jq** package, otherwise we get `jq not found` error

```suggestion
  - python ==3.9
  - jq
```

---

The existing syntax will allow it to match the latest available 3.9 release (i.e. currently 3.9.16).  The proposed syntax will lock it to 3.9.0.  I'd like to allow the flexibility for patches within 3.9.

I haven't encountered a requirement for jq in my testing, but I only tested build/install.  Where in the process did you see this error?  (Just for my understanding - fine to include an extra dep in the env specification regardless.)

---

When @frankmarch & I were building with your suggested command, we got that error.

---

To be specific, when we ran the `make`, we encountered that error.

---

Ahh, I see, it's installed system-wide on my node.  Good catch!